x-pack/solutions/observability/plugins/observability/common/constants.ts

```diff
@@ -25,12 +25,12 @@ export const observabilityAlertFeatureIds: ValidFeatureId[] = [
   AlertConsumers.SLO,
   AlertConsumers.OBSERVABILITY,
   AlertConsumers.ALERTS,
+  AlertConsumers.STACK_ALERTS,
```

Comment:
This has the side effect of enabling stack alerts to be visible in Observability rules page in both stateful and serverless. This needs to be addressed.